scripts/kind-with-registry.sh

```diff
@@ -37,7 +37,7 @@ export SERVICE_ACCOUNT_SIGNING_PUB_FILEPATH="${SERVICE_ACCOUNT_SIGNING_PUB_FILEP
 export SERVICE_ACCOUNT_SIGNING_KEY_FILEPATH="${SERVICE_ACCOUNT_SIGNING_KEY_FILEPATH:-}"
 export AZURE_CLIENT_ID_USER_ASSIGNED_IDENTITY="${AZURE_CLIENT_ID_USER_ASSIGNED_IDENTITY:-}"
 export AZURE_IDENTITY_ID_FILEPATH="${AZURE_IDENTITY_ID_FILEPATH:-$REPO_ROOT/azure_identity_id}"
-make --directory="${REPO_ROOT}" "${KUBECTL##*/}" "${KIND##*/}"
+make --directory="${REPO_ROOT}" "${KUBECTL##*/}" "${KIND##*/}" "${AZWI##*/}"
```

Comment:
Why do we need to build azwi here?

Reply:
We seem to be building all the required binaries but missing out on AZWI over there. So added it. 
It is unrelated to the aim of this PR, so happy to revert it.